picard/script.py

```diff
@@ -339,6 +339,33 @@ def register_script_function(function, name=None, eval_args=True,
     )
 
 
+def script_function(name=None, eval_args=True, check_argcount=True, prefix='func_', debug=False):
+    """Decorator helper to register script functions
+
+    It calls ``register_script_function()`` and share same arguments
+    Extra optional arguments:
+        ``prefix``: define the prefix to be removed from defined function to name script function
+                    By default, ``func_foo`` will create ``foo`` script function
+        ``debug``: if set to True, print passed parameters
+
+    Exemple:
```

Comment:
Type "Exemple" = "Example"

Reply:
That's not a typo, that's French ;) fixed & rebased